src/server/middleware/enhancers.ts

```diff
@@ -0,0 +1,38 @@
+import { Middleware } from 'koa'
+import * as uuidV1 from 'uuid/v1'
```

Comment:
Use UUIDv4 instead 🙂

Reply:
Why? :D v1 has both time and random while v4 has only random?

Reply:
uuidv1 has mac, time and a bit of random, right?

I'd argue that we do not need this added bit of uniqueness traded for 'leaking' this information (then again, this is bikeshedding omega)

Reply:
hehe, yeh maybe. I think mac adress would be the least of our concerns if our logs leak, but i see your point. expect a push in 30 seconds or so🤘

Reply:
👏